spec/yelp/endpoint/business_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:api_keys) { real_api_keys }
   let(:business) { 'yelp-san-francisco' }
   let(:client) { Yelp::Client.new(api_keys) }
+  let(:locale) { Hash[lang: 'fr'] }
```

Comment:
Given `Search` uses `locale`, do we want to stick with `locale` for consistency-sake or switch over.